plugins/InspectionGadgets/InspectionGadgetsAnalysis/src/com/siyeh/ig/migration/UnnecessaryBoxingInspection.java

```diff
@@ -38,7 +40,7 @@
 
 import javax.swing.*;
 
-public class UnnecessaryBoxingInspection extends BaseInspection {
+public class UnnecessaryBoxingInspection extends BaseInspection implements CleanupLocalInspectionTool {
```

Comment:
please no, it's not about how the code is highlighted but that it can be changed silently

Reply:
OK, rollback